doc/source/serve/http-guide.md

```diff
@@ -74,6 +74,51 @@ Existing middlewares, **automatic OpenAPI documentation generation**, and other
 Serve currently does not support WebSockets. If you have a use case that requires it, please [let us know](https://github.com/ray-project/ray/issues/new/choose)!
 ```
 
+(serve-http-streaming-response)=
+## Streaming Responses
+
+```{warning}
+Support for HTTP streaming responses is currently experimental. To enable this feature, set `RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING=1 on the cluster. If you encounter any issues, please [file an issue on GitHub](https://github.com/ray-project/ray/issues/new/choose).
+```
+
+Some applications require streaming incremental results back to the caller.
+This is common for large language models (LLMs) used for text generation.
+The full forward pass may take multiple seconds, so providing incremental results as they're available provides a much better user experience.
+
+To use HTTP response streaming, return a [StreamingResponse](https://www.starlette.io/responses/#streamingresponse) that wraps a generator from your HTTP handler.
```

Comment:
Also include setting the env variable and how you would do it in yaml.

Reply:
Set the env variable in cluster env

Reply:
Got it, definitely need to improve in 2.6. Let's document in Anyscale docs too once you're done with the ray changes.